jetty-ee9/jetty-ee9-webapp/src/main/java/org/eclipse/jetty/ee9/webapp/MetaInfConfiguration.java

```diff
@@ -169,9 +169,10 @@ public void findAndFilterContainerPaths(final WebAppContext context) throws Exce
         String classPath = System.getProperty("java.class.path");
         if (classPath != null)
         {
-            Stream.of(classPath.split(File.pathSeparator))
-                .map(resourceFactory::newResource)
-                .filter(r -> uriPatternPredicate.test(r.getURI()))
+            resourceFactory.split(classPath, File.pathSeparator)
```

Comment:
Heh, technically we shouldn't fix ee8 and ee9 because they should be bug-for-bug compatible with jetty 10/11. But ok.

Reply:
In Jetty 10/11 we ignore globs here.
The raw string is created a `File` and then if it doesn't exist, it's ignored.
The thing is, a `File` with a `*` glob always produces a `File` instance that doesn't exist (or can be opened).
But a Path cannot even be created with a `*` glob, it throws an Exception.

Reply:
understood.